backend/src/main/java/com/llm_service/llm_service/controller/conversation/ConversationController.java

```diff
@@ -32,7 +32,7 @@ public ResponseEntity<Conversation> createConversation() {
 
     @PutMapping("/{id}/continue")
     public ResponseEntity<Conversation> continueConversation(
-            @PathVariable UUID id, @RequestBody ConversationRequest conversationRequest) throws ConversationNotFound {
+            @PathVariable UUID id, @RequestBody ConversationRequest conversationRequest) throws ConversationNotFoundException {
         Conversation conversation = conversationService.update(id, conversationRequest);
 
         return ResponseEntity.status(HttpStatus.OK).body(conversation);
```

Comment:
don't we need to add this exception  `@ExceptionHandler(ConversationNotFoundException.class)` and the other one as well